### ISSUE TITLE
MQTT - Add support for setting color independantly with H, S and B

### ISSUE THREAD

**Issue by marksev1** [sticilface/Melvanimate#9]
Hi sticilface, awesome project!

Please add support for separate MQTT controls for hue, saturation and brightness (HSB).

I use HSB and am interested in that functionality, but some people might use RGB so maybe that could be an additional option.

**Comment by marksev1** [comment]
Any feedback on this?

**Comment by sticilface** [comment]
Yes. But it might be a bit complicated to do, and require quite a change to the code. I'm thinking about the best way to do it, and working on the other thing audio, reponse...
